packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {HeadingBlock, MeetingTopBarStyles} from '../MeetingTopBar'
+
+const TeamPromptHeaderTitle = styled('h1')({
+  fontSize: 16,
+  lineHeight: '24px',
+  margin: 0,
+  padding: 0
+})
+
+const TeamPromptTopBar = () => {
+  return (
+    <MeetingTopBarStyles>
+      <HeadingBlock>
+        {/* :TODO: (jmtaber129): Add back button */}
```

Comment:
do you use gitlens in vscode? 
it'll show the blame inline so you don't have to add your name if you don't want to. even lets you hover for extra details. no harm either way
![Screen Shot 2022-04-06 at 4 58 11 PM](https://user-images.githubusercontent.com/5514175/162093584-9020880a-7513-4f47-a1ff-0250f1b9a3f8.png)